modules/Classes.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable max-classes-per-file */
+/* eslint-disable no-use-before-define */
```

Comment:
- [ ] Kindly remove the line that disables `no-use-before-define` rule as requested by the previous reviewer. I get that you are permitted to disable some linter errors, but as shown in the screenshot that you shared it clearly says you can disable only linters about `multiple classes at the same file` or `unused variables` 👍